types/office-js-preview/index.d.ts

```diff
@@ -30767,7 +30767,7 @@ declare namespace Excel {
          * @remarks
          * [Api set: ExcelApi 1.12]
          */
-        selectedItems?: Array<string | PivotItem>;
+        selectedItems?: (string | PivotItem)[];
```

Comment:
build is failing but suggests keeping the original notation

```suggestion
        selectedItems?: Array<string | PivotItem>;
```

Reply:
I reported this error to the relevant internal team and will revert it here so that we can keep using the linter